File: Modul_3/Bahasa C/PRAK301-2510817120005-NorKhalisyah.c
#include <stdio.h>
int main(){
    printf("Masukkan Tiga Angka: \n");
    int a;
    printf("a = ");
    scanf("%d", &a);
    
    int b;
    printf("b = ");
    scanf("%d", &b);

    int c;
    printf("c = ");
    scanf("%d", &c);

    if (a <= b && a <=c) 
    {
        if (b < c) {
            printf ("\nHasil=  %d %d %d", a, b, c);}
        else {
            printf ("\nHasil=  %d %d %d", a, c, b);}
    }
    
    else if (b <= a && b <= c)
    {
        if (a < c){
            printf ("\nHasil=  %d %d %d", b, a, c);}
        else {
            printf ("\nHasil= %d %d %d", b, c, a);}
    }

    else 
    {
        if (a < b){
            printf ("\nHasil= %d %d %d", c, a, b);}
        else {
            printf ("\nHasil= %d %d %d", c, b, a);}
    }
return 0;
}

File: Modul_3/Bahasa Python/PRAK301-2510817120005-NorKhalisyah.py
a = int(input ("Masukkan Tiga Angka:\na = "))
b = int(input("b = "))
c = int(input("c = "))

if a <= b and a <= c:
        if b <= c:
            print(f"\nHasil= {a} {b} {c}")
        else:
            print(f"\nHasil= {a} {c} {b}")

elif b <= a and b <= c:
        if a <= c:
            print(f"\nHasil= {b} {a} {c}")
        else:
            print(f"\nHasil= {b} {c} {a}")

else:
        if a <= b:
            print(f"\nHasil= {c} {a} {b}")
        else:
            print(f"\nHasil= {c} {b} {a}")